document/chapters/02-background.tex

```diff
@@ -859,20 +859,18 @@ \subsection{Architecture-Based Acceleration}%%Status: Pending%%
 variation, the bounding interval hierarchy, might be compatible with higher
 branching ratios and thus SIMD traversal of the hierarchy \cite{Watcher_2006}.
 
-\section{Other Visualization Data Structures}
```

Comment:
surprised to see this go - do the following subsections make sense in the (prior) section that remains?

Reply:
This separation was inherited from my prelim structure when it seemed appropriate to separate SDFs from what we do and how they're used in my work, but having since read many papers on how they are used to ray trace on GPUs using ray marching I don't think they should necessarily be separated from the other data structures.

Reply:
:+1: